esx_service/vsan_policy.py

```diff
@@ -67,6 +68,7 @@ def create_policy_file(filename, content):
             f.write(content)
             f.write('\n')
     except:
+        logging.exception("Failed to open %s for writing", filename)
```

Comment:
There are a lot of places in vmdk_ops.py where exceptions are handled with no logs. Will be great to log all exceptions (findVMByName(), findChild())